tests/runtests.py

```diff
@@ -198,12 +198,22 @@ def no_available_apps(self):
     # Load all the test model apps.
     test_modules = get_test_modules()
 
+    found_start = not (start_at or start_after)
     installed_app_names = set(get_installed())
     for modpath, module_name in test_modules:
         if modpath:
             module_label = modpath + '.' + module_name
         else:
             module_label = module_name
+
+        if not found_start:
+            if start_at and (module_label == start_at or module_label.startswith(start_at + '.')):
+                found_start = True
+            elif start_after and (module_label == start_after or module_label.startswith(start_after + '.')):
```

Comment:
We use the same logic in at least three places, I would add internal hook, e.g. `_module_match_label(self, module_label, label)`.